dev-docs/best-practices/microservices.md

```diff
@@ -121,3 +121,15 @@ Be friendly and document the errors in the API's `description` property, as they
 
 *Do not use* `taskcluster-lib-stats` or `raven`.
 Instead, use `taskcluster-lib-monitor` as described in its documentation.
+
+### taskcluster-web-server
+
+#### Contents
+
+* [Schema Field Definitions](#schema-field-definitions)
+
+#### Schema Field Definitions
```

Comment:
To fit into the rest of the document, I think this will make sense as a `### GraphQL Schemas` section under `## Implementation`

Reply:
Done but I suspect there may be more changes in the future so perhaps a `taskcluster-web-server` section is more appropriate. Let me know what you think.

Reply:
Sure, that makes senes, too -- whatever fits with the flow of the document.